### ISSUE TITLE
paradox-list-packages not available

### ISSUE THREAD

**Issue by remerci** [syl20bnr/spacemacs#5877]
env: latest version of spacemacs (0.105.19) and emacs-snapshot (25.1.50.2) in ubuntu 14.04 LTS
When typeing [SPC a k] to call paradox-list-packages, the modeline shows "Required feature ‘spinner’ was not provided" and Message buffer shows:
Eager macro-expansion failure: (error "Required feature ‘spinner’ was not provided") [2 times]
progn: Required feature ‘spinner’ was not provided

**Comment by StreakyCobra** [comment]
It works as expected here. Maybe it's a Emacs25 related issue.

**Comment by TheBB** [comment]
Nobody has been able to reproduce.

**Comment by magoyette** [comment]
I got that same issue today with Emacs 24.5 on Windows 7. My Emacs config is not based on Spacemacs. It looks like it's an intermittent issue, paradox installed successfully the third time after 2 failed attempts.

**Comment by hshin23** [comment]
Have the same issue with it not being able to find a paradox package with the corresponding version name...<br/>
emacs --version == 25.1.1 <br/>
uname -r = 4.8.4-1-ARCH

**Comment by nixmaniack** [comment]
@hshin23 `paradox` recently [updated](https://github.com/Malabarba/paradox/commit/17a6690d42a1e854ec270ed930c7494077570fc8) its dependency `spinner` to version 1.7.3 which hasn't been updated on [`elpa`](https://elpa.gnu.org/packages/spinner.html) yet. 

I faced the same issue, so I copied the source of 1.7.3 and placed it inside my elpa dir as a workaround.
```
tree ~/.emacs.d/elpa/spinner-1.7.3
~/.emacs.d/elpa/spinner-1.7.3
├── spinner-autoloads.el
├── spinner.el
└── spinner-pkg.el
```

Generated autoloads file, then copied `spinner-pkg.el` from older 1.7.1 version and updated version to 1.7.3. I also had `dotspacemacs-install-packages` set to  `'used-but-keep-unused` to make sure it doesn't delete this package(might not be required but I keep it that way).
